### PR TITLE
fix: selfhosted docs

### PR DESCRIPTION
**Pull request by vinckr**
This patch updates the self-hosted section to the proposed structure (see internal notion doc)
It also moves a some of the docs buried in kratos/hydra to a higher level in self-hosted (see notion as well)

Removes the following documents or moves their path
double-check new redirects:

- https://docs-ofhcjzua1-ory.vercel.app/docs/ecosystem/deployment
- https://docs-ofhcjzua1-ory.vercel.app/docs/kratos/guides/high-availability-ha
- https://docs-ofhcjzua1-ory.vercel.app/docs/hydra/self-hosted/scaling-hydra
- https://docs-ofhcjzua1-ory.vercel.app/docs/ecosystem/logging
- https://docs-ofhcjzua1-ory.vercel.app/docs/ecosystem/versioning
- https://docs-ofhcjzua1-ory.vercel.app/docs/ecosystem/roadmap
- https://docs-ofhcjzua1-ory.vercel.app/docs/hydra/benchmark

<table>
  <tr>
    <td><h2>Before</h2>Screenshot 2023-03-24 at 09 35 33</td>
    <td><h2>Now</h2>Screenshot 2023-03-24 at 09 35 21</td>
  </tr>
</table>

### CODE REVIEW

File: docs/self-hosted/operations/15_observability.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+id: observability
+title: Observability
+---
+
+## Prometheus
+
+Ory technology exposes an endpoint for snapshot Prometheus data. This endpoint is usually found at the administrative port for Ory
```
Comment (tomekpapiernik) [inline docs/self-hosted/operations/15_observability.md:8]: > Ory technology exposes an endpoint for snapshot Prometheus data.

Unclear. What do you mean? That there is an endpoint where you can get snapshots of data collected by Prometheus?

File: docs/kratos/social-signin/06_ory.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
```
Comment (tomekpapiernik) [inline docs/kratos/social-signin/06_ory.mdx:1]: 1. This document seems to be a new doc in the "integrating social sign-in providers" section of the docs. These docs weren't supposed to be in the scope of your PR, right? 
2. The doc is not accessible through the sidebar. 
3. I don't understand why you would add an Ory Network project as a social sign-in provider to an Ory Network project - could you explain?

File: docs/self-hosted/operations/01_tracing.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+At a glance, you are able to see that:
+
+- The request failed
+- The request took ~80ms
+- It resulted in a 409
+- The hash comparison to validate the client's credentials took a whopping 70ms. Bcrypt is expensive!
+- The various database operations performed
+
+_Note: in order to see spans around database interactions, you must be using a SQL backend (i.e. MySQL or Postgres)._
```
Comment (tomekpapiernik) [inline docs/self-hosted/operations/01_tracing.md:79]:
```suggestion
:::note
 
To see spans around database interactions, you must be using a SQL backend, such as MySQL or Postgres.

:::
```

File: docs/ecosystem/upgrading.md [hunk 4]
```diff
@@ -24,3 +26,42 @@ Before upgrading to a newer version, please make sure to check with these docume
...
+
+The Ory ecosystem consists of multiple services versioned using [semantic versioning](https://semver.org). This section explains
+how we define service versions and what they mean.
+
+## Development Stages
+
+:::important
+
+**Ory only releases software that is stable and ready for production!**  
+The sandbox/incubating stage is an indicator of _how much_ the API could change in the future, including backwards incompatible
```
Comment (tomekpapiernik) [inline docs/ecosystem/upgrading.md:40]:
```suggestion
The sandbox/incubating stage is an indicator of how much the API could change in the future, including backward incompatible
```

File: docs/ecosystem/upgrading.md [hunk 5]
```diff
@@ -24,3 +26,42 @@ Before upgrading to a newer version, please make sure to check with these docume
 
 **If you haven't already subscribed to our release announcements, [subscribe now](http://eepurl.com/di390P)!** We announce
 important security releases in this newsletter.
+
+## Versioning
+
+The Ory ecosystem consists of multiple services versioned using [semantic versioning](https://semver.org). This section explains
+how we define service versions and what they mean.
+
+## Development Stages
```
Comment (tomekpapiernik) [inline docs/ecosystem/upgrading.md:35]:
```suggestion
## Development stages
```

File: src/sidebar.js [hunk 6]
```diff
@@ -369,225 +369,243 @@ module.exports = {
...
-    "ecosystem/deployment",
+    {
+      type: "category",
+      label: "Operations and SRE",
+      items: [
+        "self-hosted/operations/tracing",
+        "self-hosted/operations/api-access-control",
+        "self-hosted/operations/scalability",
+        "self-hosted/operations/observability",
+        "self-hosted/operations/logging",
```
Comment (tomekpapiernik) [inline src/sidebar.js:398]: You have all the files in a single directory and ordered through filenames - everything's in place for you to turn this into an "autogenerated" category. :)

File: src/sidebar.js [hunk 7]
```diff
@@ -369,225 +369,243 @@ module.exports = {
...
-                "hydra/self-hosted/tracing",
-                "hydra/guides/cookies",
-                "hydra/guides/cors",
-                "self-hosted/hydra/debug/csrf",
-              ],
-            },
+            "kratos/debug/troubleshooting",
+            "kratos/debug/csrf",
+            "kratos/debug/performance-out-of-memory-password-hashing-argon2",
+            "kratos/debug/debug-docker-delve-ory-kratos",
```
Comment (tomekpapiernik) [inline src/sidebar.js:448]: Another topic contained in a single dir. Update filenames to facilitate ordering and turn into an "autogenerated" category?

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]
Can you please update the docs you edited to follow the current documentation guidelines? I think it's a good opportunity to do so. :)

File: docs/kratos/social-signin/06_ory.mdx (hunk 2 above)
Comment (vinckr) [reply docs/kratos/social-signin/06_ory.mdx:1]: Oh crap yea that was not supposed to be in this PR. 
And yea there should be some explanation of the usecase of this.
But will be handled in a different PR.

**Review by tomekpapiernik — APPROVED** [review]
LGTM now, however, you didn't expand the frontmatter in one of the docs so that H4s are rendered in the ToC. That's not a big deal so I'm leaving this up to you to decide whether to implement that or not.

**Comment by vinckr** [comment]
You are right I forgot that one @tomekpapiernik 😅 

But on the other hand it looks fine in the preview? maybe it is not needed and configured globally or something? 

Screenshot 2023-03-29 at 12 21 59

**Pull request merged**